stringzilla/pybind11.cpp

```diff
@@ -16,8 +16,11 @@ typedef SSIZE_T ssize_t;
 #include <unistd.h> // `ssize_t`
 #endif
 
+#include <iostream>
```

Comment:
Why would you need this? It makes builds much slower.

Reply:
Sorry, my fault, it's not supposed to be here.

Reply:
Fixed.